### PR TITLE
Install and configure dev dependencies

### PR DESCRIPTION
**Pull request by hauptdigital**
First pull request of the project 😸 

I added the following dev dependencies to the project to improve code quality

```js
"devDependencies": {
    "eslint": "^6.8.0",
    "eslint-config-prettier": "^6.10.1",
    "eslint-plugin-prettier": "^3.1.2",
    "eslint-plugin-react": "^7.19.0",
    "husky": "^4.2.3",
    "lint-staged": "^10.1.3",
    "prettier": "^2.0.4"
  },
````

React will be added later, when I configure the front end dependencies, but I already installed `eslint-plugin-react`.

When files are staged, the husky pre-commit hook will run the lint-staged fix:

```js
"husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "*.{js,jsx}": "eslint --fix",
    "*.{js,jsx,ts,tsx,md,html,css}": "prettier --write"
  },
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hauptdigital/deepspeech-notes.git"
+  },
+  "author": "Marc Haupt",
+  "license": "ISC",
```
Comment (lmachens) [inline package.json:14]: I recommend `MIT` licence

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "deepspeech-notes",
+  "version": "1.0.0",
+  "description": "Create notes with speech-to-text technology",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (lmachens) [inline package.json:7]: I think you will add the scripts later?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "deepspeech-notes",
+  "version": "1.0.0",
```
Comment (lmachens) [inline package.json:3]: `"private": true,` is recommended to prevent publishing this to npmjs.com by mistake

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
```
Comment (lmachens) [inline .gitignore:18]: Later, this should be `build` or `client/build` if you have the default build folder in create-react-app

File: package.json (hunk 1 above)
Comment (hauptdigital) [reply package.json]: Thanks. Fixed this in commit 883eb561d2450cf4fb91edc4e57a729ac84aecbf

File: package.json (hunk 3 above)
Comment (hauptdigital) [reply package.json:3]: Correct. Fixed this in commit 883eb561d2450cf4fb91edc4e57a729ac84aecbf

File: package.json (hunk 2 above)
Comment (hauptdigital) [reply package.json:8]: Yes, after React and storybook are installed, I will add all build and test scripts.

File: .gitignore (hunk 4 above)
Comment (hauptdigital) [reply .gitignore:18]: Thanks for the info 👍

**Pull request merged**